src/EventEdition/InfoPanel.vala

```diff
@@ -264,6 +285,31 @@ public class Maya.View.EventEdition.InfoPanel : Gtk.Grid {
                 Gtk.TextBuffer buffer = new Gtk.TextBuffer (null);
                 buffer.text = property.get_comment ();
                 comment_textview.set_buffer (buffer);
+
+                // Identify links in the description and apply a tag to style them
+                var link_regex = new GLib.Regex ("https?://[\\w.-]*(\\/[\\w-\\d]*)*");
+                comment_link_tag = buffer.create_tag ("link");
+                comment_link_tag.event.connect (on_link_clicked);
+                comment_link_tag.set_property ("foreground", "#0000FF");
+                comment_link_tag.underline = Pango.Underline.SINGLE;
+                GLib.MatchInfo mi;
+
+                Gtk.TextIter search_iter;
+                buffer.get_start_iter (out search_iter);
+                if (link_regex.match (buffer.text, 0, out mi)) {
+                    do {
+                        int start, end;
+                        mi.fetch_pos (0, out start, out end);
+                        var link_text = buffer.text.substring (start, end - start);
+
+                        Gtk.TextIter starti, endi;
+                        search_iter.forward_search (link_text, Gtk.TextSearchFlags.TEXT_ONLY, out starti, out endi, null);
+
+                        buffer.apply_tag (comment_link_tag, starti, endi);
+                        search_iter.forward_chars (link_text.length);
+                    } while (mi.next ());
+                }
```

Comment:
I'm not familiar with this so please don't take my comment too seriously. Would it be possible to do the regex search on `TextBuffer.changed`? I tried your PR and when I paste links in, it only adds the tag after I've saved the event and reopened it. Note however that doing a regex search on each `TextBuffer.changed` would probably be incredibly inefficient but I am of the opinion that event descriptions are typically short and at most contain 100 words or so so performance shouldn't be too big of a hit.

Reply:
I don't think we should assume that descriptions are going to be short, especially the type that have links. Auto-generated Zoom or Webex events have very long descriptions. That said, I'm not sure what the performance hit of regex on change would be.

Two potential compromises I can think of:
1. Run the regex when the description field loses focus (so the focus goes somewhere else, like the title). This still wouldn't parse immediately, but it's possible that people could change the description and then go somewhere else, and then they'd see the benefit while they're still editing.
2. Use a timeout to wait a second or two after editing stops, then parse. I've seen some work over in elementary/code that worked with this.